.ibm/pipelines/value_files/values_showcase.yaml

```diff
@@ -11,7 +11,91 @@ global:
     # Every item defines the plugin `package` as a [NPM package spec](https://docs.npmjs.com/cli/v10/using-npm/package-spec),
     # an optional `pluginConfig` with plugin-specific backstage configuration, and an optional `disabled` flag to disable/enable a plugin
     # listed in `includes` files. It also includes an `integrity` field that is used to verify the plugin package [integrity](https://w3c.github.io/webappsec-subresource-integrity/#integrity-metadata-description).
-    plugins: []
+    plugins:
+        - package: ./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-dynamic
+          disabled: false
+          pluginConfig:
+            catalog:
```

Comment:
Do you want to explicitly overwrite the defalt `pluginConfig` already provided in the `dynamic-plugins.default.yaml` ?

(same comment in the other additions below)